File: Ejercicio 5 - Servlet/src/main/java/model/entidad/Usuario.java
package model.entidad;

public class Usuario {

	private String nombre;
	private String password;
	
	public Usuario(String nombre, String password) {
		this.nombre=nombre;
		this.password=password;
	}
	
	public String getNombre() {
		return nombre;
	}
	public String getPassword() {
		return password;
	}
	
	public void setNombre(String nombre){
		this.nombre=nombre;
	}
	public void setPassword(String password){
		this.password=password;
	}
}

File: Ejercicio 21.3 - App videojuegos con servicio login/videojuegosAngular/src/app/entidades/Usuario.ts
import { Videojuegos } from "./Videojuegos";
export class Usuario{

    private _nombre: string;
    private _password: string;
    private _listaV: Videojuegos[];
    
    constructor(nombre:string,password:string, listaV:Videojuegos[]){
        this._nombre=nombre
        this._password=password
        this._listaV=listaV
    }
    public get password(): string {
        return this._password;
    }
    public set password(value: string) {
        this._password = value;
    }
    public get nombre(): string {
        return this._nombre;
    }
    public set nombre(value: string) {
        this._nombre = value;
    }
    public get listaV(): Videojuegos[] {
            return this._listaV;
        }
    public set listaV(value: Videojuegos[]) {
        this._listaV = value;
        }
}